mappings/net/minecraft/network/PacketByteBuf.mapping

```diff
@@ -1,43 +1,468 @@
 CLASS net/minecraft/class_2540 net/minecraft/network/PacketByteBuf
+	COMMENT A packet byte buf is a specialized byte buf with utility methods adapted
+	COMMENT to Minecraft's protocol. It has serialization and deserialization of
+	COMMENT custom objects.
+	COMMENT
+	COMMENT <div class="fabric"><table border=1>
+	COMMENT <caption>Custom object handling</caption>
+	COMMENT <tr>
+	COMMENT  <th><b>Object Type</b></th> <th><b>read method</b></th> <th><b>write method</b></th>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT  <td>Codec-based</td><td>{@link #decode(Codec)}</td><td>{@link #encode(Codec, Object)}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT  <td>{@link Collection}</td><td>{@link #readCollection(IntFunction, Function)}</td><td>{@link #writeCollection(Collection, BiConsumer)}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT  <td>{@link IntList}</td><td>{@link #readIntList()}</td><td>{@link #writeIntList(IntList)}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT  <td>{@link Map}</td><td>{@link #readMap(IntFunction, Function, Function)}</td><td>{@link #writeMap(Map, BiConsumer, BiConsumer)}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT  <td>{@code byte[]}</td><td>{@link #readByteArray()}</td><td>{@link #writeByteArray(byte[])}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT  <td>{@code int[]}</td><td>{@link #readIntArray()}</td><td>{@link #writeIntArray(int[])}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT  <td>{@code long[]}</td><td>{@link #readLongArray()}</td><td>{@link #writeLongArray(long[])}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT  <td>{@link BlockPos}</td><td>{@link #readBlockPos()}</td><td>{@link #writeBlockPos(BlockPos)}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT  <td>{@link ChunkPos}</td><td>{@link #readChunkPos()}</td><td>{@link #writeChunkPos(ChunkPos)}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT  <td>{@link ChunkSectionPos}</td><td>{@link #readChunkSectionPos()}</td><td>{@link #writeChunkSectionPos(ChunkSectionPos)}</td>
```

Comment:
This was proguard removed before, so you should probably check javadoc build info and see if there are similar dead links